terraform/shared.tfvars

```diff
@@ -1,5 +1,6 @@
 region = "us-east-1"
 vpc_id = "vpc-097c92bf21a7082e9" # 172.31.0.0/16
+subnet = "172.31.0.0/16"
```

Comment:
This should be removed - we already know the subnets via availability_zone_to_subnet_map